### PR TITLE
Don't use Backbone to define masthead elements.

### PR DESCRIPTION
**Pull request by jmchilton**

### CODE REVIEW

File: client/galaxy/scripts/layout/menu.js [hunk 1]
```diff
@@ -34,315 +42,301 @@ export function logoutClick() {
...
+}
+
+function loadWebhookMenuItems() {
+    //
+    // Webhooks
+    //
+    Webhooks.load({
+        type: "masthead",
+        callback: function (webhooks) {
+            $(document).ready(() => {
```
Comment (OlegZharkov) [inline client/galaxy/scripts/layout/menu.js:320]: not a big deal and I guess wouldn't worth time, if it doesn't just work out of the box... But with ``` document.addEventListener('DOMContentLoaded', ()=>{} );``` we might be able to get rid of ```import $ from "jquery";``` in this file

Comment (jmchilton) [reply client/galaxy/scripts/layout/menu.js:320]: I will make this change, thanks!

Comment (jmchilton) [reply client/galaxy/scripts/layout/menu.js:320]: Change included in #9822 https://github.com/galaxyproject/galaxy/pull/9822/commits/7aa10bca2c14c2f59c40dc2f04424bc087a18e73.

Comment (OlegZharkov) [reply client/galaxy/scripts/layout/menu.js:320]: Great!

**Pull request merged**